src/pages/SingleVideoPage/index.jsx

```diff
@@ -1,10 +1,44 @@
-import React from "react";
+import React, { useEffect } from "react";
+import { Link, useLocation } from "react-router-dom";
 import Layout from "../../components/Layout";
-
+import Loader from "../../components/Loader";
+import { useVideos } from "../../context/VideosContext";
+import RelatedVideos from "./RelatedVideos";
+import VideoInfo from "./VideoInfo";
+import "./SingleVideoPage.css";
 const SingleVideoPage = () => {
+  const location = useLocation();
+  const { fetchVideoInfo, video } = useVideos();
+  useEffect(async () => {
```

Comment:
As far as I know async should not be used in useEffect like this. Check [this](https://devtrium.com/posts/async-functions-useeffect)

Reply:
Will fix